components/widgets/menu/menu.tsx

```diff
@@ -116,9 +118,9 @@ export default class Menu extends React.PureComponent {
                 className='a11y__popup Menu'
                 id={id}
                 role='menu'
+                ref={this.node}
```

Comment:
You are moving the ref from one node to another changing the behavior here, you need to keep the ref where it was before, or change the usage of the ref in the rest of the code.

Reply:
I moved it while doing tests, and for some reason I thought it was necessary to move it. On 
1f1f17d I moved it back to the ul element. Thanks!